package.json

```diff
@@ -18,6 +18,7 @@
         "pretest": "npm run compile",
         "compile": "npm run clean && tsc -p ./",
         "composer": "composer install",
+        "download-phpactor": "curl -L https://github.com/phpactor/phpactor/releases/download/2023.09.24.0/phpactor.phar > phpactor.phar",
```

Comment:
how is this going to get updated?

Reply:
you could use `releases/latest/download`?

Reply:
Renovate will make PRs for updating versions. Should be a fixed version so we have consistent and repeatable builds. I'll merge this after renovate setup.